webpack.config.js

```diff
@@ -26,7 +30,13 @@ const config = {
       filename: './dist/wp-bookings-front-ui.css',
       allChunks: true,
     }),
+    new webpack.DefinePlugin({
+      BOWER_DEPS: JSON.stringify(Object.keys(bowerConfig.dependencies).reduce((result, key) => {
+        result[key] = bowerConfig.dependencies[key].split('.').slice(0, -1).join('.')
```

Comment:
What's the purpose of this splitting and re-joining?

Reply:
This is the way to remove extension of file.